lib/editor/actions/trip.js

```diff
@@ -122,8 +122,8 @@ export function fetchTripsForCalendar (
               stopSequence: stop_sequence
               arrivalTime: arrival_time
               departureTime: departure_time
-              startPickupDropoffWindow: start_pickup_dropoff_window
-              endPickupDropoffWindow: end_pickup_dropoff_window
+              startPickupDropoffWindow: start_pickup_drop_off_window
```

Comment:
It does look a bit silly doesn't it with the lower case o... I'll leave it up to the second reviewer to decide